components/builder-web/stylesheets/base/_forms.scss

```diff
@@ -75,7 +75,7 @@ select[multiple] {
   box-sizing: border-box;
   font-family: $base-font-family;
   font-size: rem(14);
-  margin-bottom: $base-spacing * 1.5;
+  margin-bottom: $small-spacing * 2;
   padding: $small-spacing $small-spacing * 1.5;
```

Comment:
By reducing the margin below form inputs, we reduce the space to validation results (text) as well. The validation text is not consistently marked up in all areas where it appears, so this felt like the simplest fix.